my_list.rb

```diff
@@ -0,0 +1,36 @@
+require './enumerable'
```

Comment:
Great job team 🤝

- [ ] Kindly use `require_relative` instead of `require` to import a file. This is a ruby convention.